README.md

```diff
@@ -77,6 +79,9 @@ Then, all the pieces should be up!
 * All manager nodes of your Wazuh manager cluster should be reachable at wazuh-manager.some-domain.com:1514
 * Kibana and the Wazuh Kibana application should be available at https://wazuh.some-domain.com:443
 
+## Wazuh agents deployment
+This repository does not show how to deploy the Wazuh agent in a Kubernetes cluster. Normally, we would use a DaemonSet to deploy the agent on each Kubernetes node. To do that, we would need a Docker image with the Wazuh agent installed on it and then we would need to mount almost every folders of the host inside that container (`/bin`, `/etc`, `/var/log`, etc.). It would be a very complicated task since you cannot simply mount the `/bin` folder of you host in the `/bin` folder of your container. Therefore, creating such Docker image an using it in a Kubernetes DaemonSet is not the ideal way to deploy a Wazuh agent. Instead, you should take a look at the [Wazuh Ansible playbooks project](https://github.com/wazuh/wazuh-ansible) or at the [Wazuh Puppet module project](https://github.com/wazuh/wazuh-puppet) to deploy your Wazuh agents.
```

Comment:
almost every folder

... folder of your host ...

creating such Docker images and using them in . . .

Reply:
"creating such Docker images and using them in"  is not the intent here. You only need one image, you don't need many. :)